app/Models/Multiplayer/Room.php

```diff
@@ -462,6 +462,7 @@ public function startGame(User $host, array $rawParams)
             'playlist:array',
             'type',
             'queue_mode',
+            'auto_start_duration:int',
```

Comment:
Is this field going to be optional?
Does `0` mean "not set" or "start immediately"?

Reply:
0 means no auto